activerecord/lib/active_record/serialization.rb

```diff
@@ -9,7 +9,7 @@ module Serialization
     end
 
     def serializable_hash(options = nil)
-      options = options.try(:clone) || {}
+      options = options.try(:dup) || {}
```

Comment:
I just realized that if you change the signature to `def serializable_hash(**options)`, and that is called _without_ splatting a hash being passed in, it will modify the hash. So that is probably not a viable option.